demos/projects/NXP/mimxrt1060/main.c

```diff
@@ -241,6 +244,12 @@ void CRYPTO_InitHardware( void )
 }
 /*-----------------------------------------------------------*/
 
+bool xAzureSample_IsConnectedToInternet()
+{
+    return true;
```

Comment:
For clarity, can you add a comment saying this isn't implemented for now in the samples where it isn't?

Reply:
Makes sense. Done.